test/utils/safeImageUrl.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Safe Image Url
+ *
+ * When using dummy image urls via faker in an img tag, if the
+ * component you're testing gets mounted karma will try to load that url.
+ * This makes a call out to the internet, which is unnecessary and seems to
+ * always crash phantomjs.
+ *
+ * Karma is configured to serve images locally from the /test/images folder,
+ * see the documentation for more details:
+ * https://github.com/karma-runner/karma/blob/master/docs/config/02-files.md#loading-assets
+ */
+
+/**
+ * Currently always returns the same one image but could be updated to
+ * randomize if we want to add more images to that folder.
+ *
+ * @returns {string} local url
+ */
+const safeImageUrl = () => '/base/test/images/logo.png'
```

Comment:
That doesn't work. Looking for the non-existent URL returns a 404 and crashes phantomjs:
```
    active item
      ✔ defaults to no result active
      ✔ defaults to the first item with selectFirstResult
      ✔ moves down on arrow down when open
19 09 2016 10:41:45.317:WARN [web-server]: 404: /foo.png
      ✔ moves up on arrow up when open
      ✔ skips over items filtered by search
      ✔ scrolls the selected item into view
      ✔ closes the menu
      ✔ uses custom renderer
    category
      ✔ defaults to the first item with selectFirstResult
      ✔ moves down on arrow down when open
19 09 2016 10:41:45.380:ERROR [launcher]: PhantomJS crashed.
	
19 09 2016 10:41:45.382:ERROR [launcher]: PhantomJS failed 2 times (crashed). Giving up.
```